templates/amp-paired-browsing.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * AMP Paired Browsing experience template.
+ *
+ * 🚫🚫🚫
+ * DO NOT EDIT THIS FILE WHILE INSIDE THE PLUGIN! Changes You make will be lost when a new version
+ * of the AMP plugin is released.
+ * 🚫🚫🚫
```

Comment:
This comment isn't relevant here because it should not be overridden by any theme.